cmd/wsh/cmd/wshcmd-file.go

```
@@ -58,140 +87,114 @@ func init() {
+	Example: "  wsh file cp wavefile://block/config.txt ./local-config.txt\n  wsh file cp ./local-config.txt wavefile://block/config.txt\n  wsh file cp wsh://user@ec2/home/user/config.txt wavefile://client/config.txt",
 	Args:    cobra.ExactArgs(2),
 	RunE:    activityWrap("file", fileCpRun),
 	PreRunE: preRunSetupRpcClient,
 }
 
-func fileCatRun(cmd *cobra.Command, args []string) error {
-	ref, err := parseWaveFileURL(args[0])
-	if err != nil {
-		return err
-	}
+var fileMvCmd = &cobra.Command{
```

golangci-lint (1.62.2)

var `fileMvCmd` is unused

(unused)